jobs/pdns/spec

```diff
@@ -7,16 +7,21 @@ templates:
   pipe.conf.erb: etc/pipe.conf
   pipe.erb: bin/pipe
   pipe.example.com.pl: bin/pipe.example.com.pl
+  migrations/gsqlite3.sql: migrations/gsqlite3.sql
   post-start.erb: bin/post-start
   pre-start.erb: bin/pre-start
 
-
 packages:
 - boost
 - pdns
 - sqlite
 
 properties:
+  mode:
```

Comment:
Any reason to use this `mode` property? Seems you could do the logic in the pre-start with the `sqlite.database` property. Also, mode is part of the pdns.conf which is entirely provided, not templated, so not sure where else you would use this property.

Reply:
Updated.